.github/workflows/diff-change-to-dist.yaml

```diff
@@ -33,8 +33,16 @@ jobs:
       - name: Generate diff
         id: diff
         run: |
-          git diff -M05 origin/$GITHUB_BASE_REF -- dist \
-            > $GITHUB_WORKSPACE/dist.diff
+          # Using `origin/$GITHUB_BASE_REF` to avoid actually checking out the branch
+          # as all we need is to let Git diff the two references
+          .github/workflows/scripts/dist-diff.sh origin/$GITHUB_BASE_REF $GITHUB_WORKSPACE
```

Comment:
Should we consolidate all scripts into [`./bin`](https://github.com/alphagov/govuk-frontend/tree/main/bin)?

Thinking ahead to when we do releases from GitHub Actions